### DISCUSSION TITLE
Improving the Startup Tie of the Program after Idle Time or Opening for the First Time

### DISCUSSION THREAD

**Discussion by Samermaan** [d2phap/ImageGlass#1985] [General]
Hi, 
I really think this program is awsome and one of the best! 
My only question is that if there is a way to increase the start speed of the program after some idle time of not using the program. 
Even if there is an instance/window that is currently open and minimized, opening a new image file/instance would take much longer time when the program is not used for some time and new image file is opened. 
Usually in other windows software, I keep an instace/window open of that software and this does increase the speed of loading a new file/instance using the same program. 
In ImageGlass, It didnt seem the same case. 
I tried to automatically increase the CPU priority to high and use some windows software to boost start time but didnt have much luck with ImageGlass. 
Other than the above, the software is awsome.
I tried the program and the windows app versions and saw about the same start time after being idle. 
I already have selected the "Enable Startup Boost"    
Can you please help in this? 

Thanks,
Sam

**Comment by d2phap** [comment]
Hi @Samermaan

Thanks for sharing the issue.
This is the common issue with apps built with WinForms .NET. Enabling "Startup Boost" will reduce the startup time, but if this method does not help much on your machine, there is currently no other way.

Optimizing for the startup time is always in my list, a lot of work have done in v9.1 to make it launch faster than v9.0, but apparently, it's not enough. I will continue to improve it.

**Reply by Samermaan** [reply]
Hi @d2phap,

Thanks for the clarification. I see the latest version is running on .NET Runtime 8.0
Knowing that Microsoft released NET Runtime 9.0 which people say it much faster than 8.0, will newer Imageglass version will be based on NET 9.0? 

Thanks,
Sam
